tasks-by-users/Naufil-Shemle/03-css-positioning/index.html

```diff
@@ -0,0 +1,119 @@
+            quidem, adipisci modi fugit molestiae voluptatem voluptas excepturi
+            ab in molestias hic debitis cum eos cupiditate vero sequi suscipit
+            expedita. Neque iste ipsa optio nobis dolorem earum laboriosam omnis
+            labore voluptas ab voluptatum, impedit qui ipsum vel eligendi
+            delectus eum quo, repudiandae minima distinctio. Illo nesciunt quae
+            veritatis, ducimus blanditiis velit sunt!
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui
+            reprehenderit non optio reiciendis ea doloremque recusandae possimus
+            debitis quidem natus dicta id esse iusto eveniet expedita cum vel
+            molestias officia, ex consectetur perspiciatis unde sunt perferendis
+            inventore? Itaque corporis ipsa, necessitatibus debitis est
+            molestias aliquam minus error quasi eaque consequuntur odio nam
+            dolores nesciunt veniam accusantium unde quam dicta. Quis id
+            accusantium laboriosam. Commodi hic facilis et nesciunt, totam
+            mollitia ullam quam dolorem ratione explicabo porro. Dolorum nostrum
+            quidem, adipisci modi fugit molestiae voluptatem voluptas excepturi
+            ab in molestias hic debitis cum eos cupiditate vero sequi suscipit
+            expedita. Neque iste ipsa optio nobis dolorem earum laboriosam omnis
+            labore voluptas ab voluptatum, impedit qui ipsum vel eligendi
+            delectus eum quo, repudiandae minima distinctio. Illo nesciunt quae
+            veritatis, ducimus blanditiis velit sunt!
+          </p>
+        </section>
+        <aside>
+          <h2>This is column 2</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui
+            reprehenderit non optio reiciendis ea doloremque recusandae possimus
+            debitis quidem natus dicta id esse iusto eveniet expedita cum vel
+            molestias officia, ex consectetur perspiciatis unde sunt perferendis
+            inventore? Itaque corporis ipsa, necessitatibus debitis est
+            molestias aliquam minus error quasi eaque consequuntur odio nam
+            dolores nesciunt veniam accusantium unde quam dicta. Quis id
+            accusantium laboriosam. Commodi hic facilis et nesciunt, totam
+            mollitia ullam quam dolorem ratione explicabo porro. Dolorum nostrum
+            quidem, adipisci modi fugit molestiae voluptatem voluptas excepturi
+            ab in molestias hic debitis cum eos cupiditate vero sequi suscipit
+            expedita. Neque iste ipsa optio nobis dolorem earum laboriosam omnis
+            labore voluptas ab voluptatum, impedit qui ipsum vel eligendi
+            delectus eum quo, repudiandae minima distinctio. Illo nesciunt quae
+            veritatis, ducimus blanditiis velit sunt!
+          </p>
+          <img src="cloud image.jpg" alt="Cloud image" width="100%" />
```

Comment:
Remove inline styling @Naufil-Shemle

Reply:
Done

Reply:
Inline styling still there, there is no recent commit for this. why you resolved?? @Naufil-Shemle

Reply:
just going commit brother
wait 1 minute

Reply:
Always Resolve conversation after you made changes suggested by a reviewer. @Naufil-Shemle

Reply:
OK Sir